### PR TITLE
Blog changes

### PR DESCRIPTION
**Pull request by Heulitig**

### CODE REVIEW

File: blog/external-css.ftd [hunk 1]
```diff
@@ -29,51 +22,59 @@ published-on: November 3, 2023 at 10:27 am
...
 -- ds.blog-page:
 meta: $meta
 
--- ds.h1: Is external CSS supported?
-
-Yes, fastn supports the use of external CSS which we will talk about in the
-later sections.
-
--- ds.h1: Why do we need external CSS?
+-- ds.h3: **When external CSS should be used?**
```
Comment (Arpita-Jaiswal) [inline blog/external-css.ftd:31]: Is `ds.h3` not already bold?

File: blog/external-css.ftd [hunk 2]
```diff
@@ -29,51 +22,59 @@ published-on: November 3, 2023 at 10:27 am
...
-the latest CSS (Cascading Style Sheets) properties is crucial to make the most
-out of it. These CSS properties keeps on changing with time
-while certain properties become deprecated, newer, more advanced ones emerge.
-Some of these emerging properties are released in experimental stages, while
-some of them could be very newly released raising questions about their
-compatibility with fastn. In such scenarios, the importance of external CSS
-becomes evident. In this blog, we will see how we can make use of external CSS
-in fastn.
+In this blog, we will break down all the necessary details we
+need to know before using external CSS properties on our fastn components.
```
Comment (Arpita-Jaiswal) [inline blog/external-css.ftd:26]: Why `index-meta` and `meta` both used?

File: ftd/built-in-types.ftd [hunk 3]
```diff
@@ -20,53 +20,53 @@ These types can be categorized into two main groups: **Primitive** and
...
-  - [`ftd.mask`](built-in-types#ftd-mask)
-  - [`ftd.spacing`](built-in-types#ftd-spacing)
-  - [`ftd.text-align`](built-in-types#ftd-text-align)
-  - [`ftd.text-input-type`](built-in-types#ftd-text-input-type)
-  - [`ftd.text-style`](built-in-types#ftd-text-style)
-  - [`ftd.text-transform`](built-in-types#ftd-text-transform)
-  - [`ftd.type`](built-in-types#ftd-type)
-  - [`ftd.white-space`](built-in-types#ftd-white-space)
-  - [`ftd.type-data`](built-in-types#ftd-type-data)
+- [Primitive Types](/built-in-types#primitive-types)
```
Comment (Arpita-Jaiswal) [inline ftd/built-in-types.ftd:23]:
```suggestion
- [Primitive Types](/built-in-types/#primitive-types)
```

File: blog/external-css.ftd (hunk 2 above)
Comment (Heulitig) [reply blog/external-css.ftd]: That were initial values, now those values are different

File: blog/external-css.ftd (hunk 1 above)
Comment (Heulitig) [reply blog/external-css.ftd:34]: Fixed

File: ftd/built-in-types.ftd (hunk 3 above)
Comment (Heulitig) [reply ftd/built-in-types.ftd:23]: That extra slash was causing the reload issue when it was already an intra-link.

File: blog/breakpoint.ftd [hunk 4]
```diff
@@ -9,74 +9,60 @@ published-on: November 3, 2023 at 3:40 pm
...
 
-Browser breakpoint widths are used in web design to define how a
-webpage should be displayed and styled based on the width of the
-user's browser window or device screen. These breakpoints help create
-responsive web designs that adapt to different screen sizes and
-orientations, making websites more accessible and user-friendly
-on a variety of devices, such as desktop computers, smartphones etc.
+Today, in this blog we will see how fastn allows the use of user-defined
+custom breakpoints. Although currently we can only modify the
+default breakpoint when using ftd.document. We will cover the mentioned
```
Comment (amitu) [inline blog/breakpoint.ftd:14]: Use `backticks` when using technical terms. Also link to `ftd.document` documentation.

**Comment by Arpita-Jaiswal** [comment]
Add blog in notification too @Heulitig

**Pull request merged**